### ISSUE TITLE
After country update, Formatted number becomes empty.

### ISSUE THREAD

**Issue by ashah2193** [bl00mber/react-phone-input-2#322]
Hello,

When we have disableCountryCode set to true and we change country, the formatted number becomes empty. 

Is it a required functionality or is it a bug?
 

![image](https://user-images.githubusercontent.com/27024877/94418083-2fd79a80-0181-11eb-8299-f397c926e41e.png)

**Comment by bl00mber** [comment]
That's true for both `disableCountryCode` = false and true. The number part is skipped in both cases. You may want to change this and send a PR which will add consideration of formatted number when country is changed programmatically.
